astroquery/mast/core.py

```diff
@@ -1493,85 +1496,70 @@ def _shib_download_curl_script(self, products, out_dir):
                           "URL": [url]})
         return manifest
 
+    @deprecated(since="11/9/18", alternative="enable_s3_dataset")
     def enable_s3_hst_dataset(self):
+        return self.enable_s3_dataset()
+
+    def enable_s3_dataset(self):
         """
-        Attempts to enable downloading HST public files from S3 instead of MAST.
+        Attempts to enable downloading public files from S3 instead of MAST.
         Requires the boto3 library to function.
         """
         import boto3
         import botocore
         self._boto3 = boto3
         self._botocore = botocore
 
-        log.info("Using the S3 HST public dataset")
+        log.info("Using the S3 STScI public dataset")
         log.warning("Your AWS account will be charged for access to the S3 bucket")
         log.info("See Request Pricing in https://aws.amazon.com/s3/pricing/ for details")
         log.info("If you have not configured boto3, follow the instructions here: "
                  "https://boto3.readthedocs.io/en/latest/guide/configuration.html")
 
+    @deprecated(since="11/9/18", alternative="disable_s3_dataset")
     def disable_s3_hst_dataset(self):
+        return self.disable_s3_dataset()
+
+    def disable_s3_dataset(self):
         """
-        Disables downloading HST public files from S3 instead of MAST
+        Disables downloading public files from S3 instead of MAST
         """
         self._boto3 = None
         self._botocore = None
 
+    @deprecated(since="11/9/18", alternative="get_s3_uris")
```

Comment:
please use an international date format, preferably ISO (YYYY-MM-DD), this it super confusing.
or just add v0.3.9 for the release, I think we need to tag a version sometimes soon.

Reply:
Fixed :+1: